packages/theme-language-server-common/src/completions/CompletionsProvider.ts

```diff
@@ -49,7 +49,7 @@ export class CompletionsProvider {
 
     this.providers = [
       new HtmlTagCompletionProvider(),
-      new HtmlAttributeCompletionProvider(),
+      new HtmlAttributeCompletionProvider(documentManager),
```

Comment:
I won't have to pass in the `documentManager` once this change is introduced
https://github.com/Shopify/theme-tools/pull/507/files#diff-d0fc601312d376218785407c3e38c31fda0c22b4adec2702bdc3aee7ff24eb92R34

Reply:
507 is merged if you want to rebase and do that. But honestly not a big deal.